.github/workflows/csharp.yml

```diff
@@ -62,7 +62,7 @@ jobs:
             - name: Install shared software dependencies
               uses: ./.github/workflows/install-shared-dependencies
               with:
-                  os: ${{ matrix.os }}
+                  os: ${{ contains(matrix.os, 'macos') && 'macos' || 'ubuntu' }}
```

Comment:
```suggestion
                  os: ${{ contains(matrix.os, 'macos') && ('macos' || 'ubuntu') }}
```

Reply:
nonono, it is actually
```
os: ${{ (contains(matrix.os, 'macos') && 'macos') || 'ubuntu' }}
```
do you want brackets?

Reply:
oh yes!  please!!!!